Fix `add_electrodes` behavior when writing multiple recordings with mismatched metadata (#558)
catalystneuro/neuroconv: CHANGELOG.md

* Added deepcopy for metadata in `make_nwbfile_from_metadata`. [PR #545](https://github.com/catalystneuro/neuroconv/pull/545)

> * Alter `add_electrodes` behavior when extra property descriptions are present in metadata so that no error is thrown. [PR #558](https://github.com/catalystneuro/neuroconv/pull/558)

Reviewer: ```suggestion
* Alter `add_electrodes` behavior,  no error is thrown if a property is present in the metadata but not in the recoders. This allows to add recording objects that have different properties [PR #558](https://github.com/catalystneuro/neuroconv/pull/558)
```

I think this is is a better description but your is fine as well. Your call if you want to change it.

> * Alter `add_electrodes` behavior,  no error is thrown if a property is present in the metadata but not in the recoders. This allows to add recording objects that have different properties [PR #558](https://github.com/catalystneuro/neuroconv/pull/558)

Reviewer 2: ```suggestion
* Alter `add_electrodes` behavior,  no error is thrown if a property is present in the metadata but not in the recording extractors. This allows the combination of recording objects that have different properties. [PR #558](https://github.com/catalystneuro/neuroconv/pull/558)
```

typo
